documentation/build.md

```diff
@@ -42,16 +42,20 @@ testsuite is
   available.
 </a>
 
-The image has been successfuly tested on:
+Supported hosts: Linux`, `Windows` and `MacOS`.
+Supported hypervisors: `VirtualBox` and `libvirt`.
 
-* vagrant v2.1.1
-* VirtualBox v5.2.10
+The image has been successfuly tested using:
+
+* VirtualBox: v5.2.10
+* vagrant: v2.1.1
 
 First install the needed vagrant plugins:
 
 {% highlight shell %}
```

Comment:
I suggest removing all this section as it's possible to not need any plugin: https://github.com/skydive-project/skydive/pull/1072